src/index.js

```diff
@@ -1,3 +1,4 @@
+//#region > Imports
```

Comment:
This import region is messed up, only add a newline between internal and external imports and place external imports above internal.

Reply:
We split imports per header and a new line.